packages/strapi-design-system/src/v2/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex } from '../../Flex';
+import { VisuallyHidden } from '../../VisuallyHidden';
+import { Crumb } from './Crumb';
+import { CrumbLink } from './CrumbLink';
+
+export const Breadcrumbs = ({ label, children, ...props }) => {
+  return (
+    <Flex {...props}>
+      <VisuallyHidden>{label}</VisuallyHidden>
+      <ol aria-hidden={true}>{children}</ol>
+    </Flex>
+  );
+};
+
+const crumbType = PropTypes.shape({ type: PropTypes.oneOf([Crumb, CrumbLink]) });
+
+Breadcrumbs.displayName = 'Breadcrumbs';
```

Comment:
What do you think about defining the const first and export it later, to avoid the duplication of the name?

Reply:
Are we? Then never mind ...